tests/test_api.py

```diff
@@ -1,14 +1,18 @@
 import timereport.api as api
 
-from tests.test_data import data as event
+import tests.test_data as test_data
 import os
 
-def test_factory():
+def test_factory(event):
     # set os.environ SLACK_TOKEN
     os.environ["SLACK_TOKEN"] = api.config['SLACK_TOKEN']
     api.lambda_handler(event, None)
 
 
 
 
-test_factory()
\ No newline at end of file
+# run test_data.add
+test_factory(test_data.add)
```

Comment:
These are not proper tests. We should merge #21 first and then continue to write tests

Reply:
of course they are not proper tests. We need to assert specific functions

Reply:
So what's the point of them then?

Reply:
we need to run list and add actions
at this time without having to change the test_data each time and since there is no mechanism for doing proper tests at this time, this is the solution

Reply:
If you rebase  on to `factory` branch I will help you write proper tests instead.